pkg/apis/ceph.rook.io/v1/types.go

```diff
@@ -278,6 +278,10 @@ type CephVersionSpec struct {
 	// Whether to allow unsupported versions (do not set to true in production)
 	// +optional
 	AllowUnsupported bool `json:"allowUnsupported,omitempty"`
+
+	// ImagePullPolicy describes a policy for if/when to pull a container image
+	// One of Always, Never, IfNotPresent.
+	ImagePullPolicy string `json:"imagePullPolicy,omitempty"`
```

Comment:
I don't think this API change is needed. We are reading the pull policy from the operator config map.